backend/app/Repositories/EloquentButtonVisitorsRepository.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+
+namespace App\Repositories;
+
+use App\Entities\Visitor;
+use App\Repositories\Contracts\ButtonVisitorsRepository;
+
+class EloquentButtonVisitorsRepository implements ButtonVisitorsRepository
+{
+    public function getVisitorsCount(string $startData, string $endData, int $websiteId, int $userId): int
```

Comment:
why don't you pass DatePeriod instead of $startData and $endData? And data means different than date.